### PR TITLE
add education page

### PR DESCRIPTION
**Pull request by bgmarx**
add education page

### CODE REVIEW

**Comment by starbelly** [comment]
@bgmarx were you wanting to merge this now?

**Comment by bgmarx** [comment]
no rush. I wonder if it makes sense to break it into education / adoption

**Comment by starbelly** [comment]
@bgmarx that's what @desmondmonster was suggesting.

File: lib/erlef_web/templates/page/wg.html.eex [hunk 1]
```diff
@@ -1,52 +1,59 @@
-    <div class="container">
```
Comment (starbelly) [inline lib/erlef_web/templates/page/wg.html.eex]: Excellent tidy up job! 👍

**Comment by starbelly** [comment]
@bgmarx Could you copy/pasta the content from https://erlef.org/education-training-adoption/ into a template to wrap this up?

**Pull request merged**